### PR TITLE
XRDDEV-2025 Add members without configured encryption keys to Diagnos…

### PR DESCRIPTION
**Pull request by enelir**
…tic page Message log archive encryption table

### CODE REVIEW

**Comment by andresrosenthal** [comment]
[rerun-ci]

**Review by ricardas-buc — APPROVED** [review]

**Comment by petkivim** [comment]
This issue is scoped to version `7.1.0` so it should be merged into `beta-7.1.0` branch.

**Comment by petkivim** [comment]
The icon that indicates that the default encryption key is used should be gray when the element is disabled (=message log add-on is disabled):

Screenshot 2022-06-08 at 7 05 25

Just like the encryption status icon is grey in the screenshot.

File: src/proxy/src/main/java/ee/ria/xroad/proxy/ProxyMain.java [hunk 1]
```diff
@@ -238,7 +240,23 @@ private static void createServices() throws Exception {
...
+            return ServerConf.getMembers().stream()
+                    .filter(ProxyMain::isMemberRegistered)
+                    .collect(Collectors.toList());
+        } catch (Exception e) {
+            log.error("Failed to get members from server configuration", e);
+            throw e;
+        }
+    }
+
+    private static boolean isMemberRegistered(ClientId member) {
```
Comment (petkivim) [inline src/proxy/src/main/java/ee/ria/xroad/proxy/ProxyMain.java:257]: This filter can be removed. The list should include all local clients regardless of their status. There's no need to restrict the list to registered clients only.

**Comment by enelir** [comment]
JIRA: https://jira.niis.org/browse/XRDDEV-2025

**Review by petkivim — APPROVED** [review]
Looks good to me.

**Pull request closed without merging**